README.adoc

```diff
@@ -365,15 +354,14 @@ the pod for the GET request that you just ran:
 
 [role='command']
 ```
-kubectl exec -it [pod-name] /opt/ol/wlp/bin/server pause defaultServer
+kubectl exec -it [pod-name] /opt/ol/wlp/bin/server pause
 ```
 
 Repeat the GET request. You see the same `session-id`
 but a different `pod-name` because the session data is cached but the request
 is served by a different pod (server).
 
-Verify that the Hazelcast cluster is running by checking the Open Liberty log.
-Run the following command:
+Verify that the Hazelcast cluster is running by checking the Open Liberty log. Run the following command:
```

Comment:
Change
"Verify that the Hazelcast cluster is running by checking the Open Liberty log. Run the following command:"
To
"Verify that the Hazelcast cluster is running by checking the Open Liberty log. To check the log, run the following command:"